docs/usage/PRODUCT_SYNC.md

```diff
@@ -24,6 +24,9 @@ Utility which provides API for building CTP product update actions and product s
 1. The sync expects a list of non-null `ProductDraft` objects that have their `key` fields set to match the
 products from the source to the target. Also the target project is expected to have the `key` fields set,
 otherwise they won't be matched.
+
+**NOTE: PLEASE MAKE SURE THE `SKU` FIELDS OF ALL PRODUCTS ARE SET AS THE SYNC LIBRARY WILL BE MIGRATED TO MATCH PRODUCTS BY `SKU` INSTEAD OF `KEY` IN THE FUTURE.**
+
```

Comment:
We should validate for SKU existence and throw an error, wdyt?

Reply:
Agree, but I will then push this PR and issue #201  to the next release (to be able to provide tests for the validation and so on)

Reply:
@butenkor https://github.com/commercetools/commercetools-sync-java/pull/200/files#diff-42c36b0300af86525a087593ce1dc364R232